CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# JSON Schema
```

Comment:
As a minor change from the wiki page, the title could be more specific. Something like `JSON Schema contributor guidelines`.